### DISCUSSION TITLE
VS not publishing to Github

### DISCUSSION THREAD

**Discussion by apulido08** [community/community#137239] [Repositories] [answered]
### Select Topic Area

Question

### Body

Is their anyway to fix this issue? GIt hub works at my school computer but some odd reason it wont work at my home pc. i have noticed that my school work has a different username than my home one

**Comment by Yackens** [comment]
Compare git config --global user.name and user.email on both PCs. Match them using:

```
git config --global user.name "School Username"
git config --global user.email "your.email@school.com"
```

If you found this answer useful, please, mark my answer as the correct one to help me obtain GitHub achievement: Galaxy Brain 😄

**Comment by ThisIs-Developer** [comment, marked as answer]
### Resolving GitHub Issues on Home PC

1. **Check Git Configuration:** Verify your Git username and email:
   ```bash
   git config --global user.name "YourUsername"
   git config --global user.email "your.email@example.com"
   ```
2. **Check SSH Keys:** Ensure SSH keys are set up correctly:
   ```bash
   ssh -T git@github.com
   ```
### Deleting GitHub Credentials from Windows Credential Manager

1. **Open Credential Manager:** Press `Windows + R`, type `control`, and open **Credential Manager**.
2. **Access Windows Credentials:** Click on **Windows Credentials**.
3. **Find and Remove GitHub Credentials:** Locate entries for GitHub, expand, and click **Remove**.
4. **Confirm Deletion:** Click **Yes** if prompted.

After deleting, you'll be prompted for your GitHub username and password the next time you connect.

### If this helps, please mark as answer!😴
